internal/pkg/zebra/zapi.go

```diff
@@ -35,6 +36,11 @@ const (
 	INTERFACE_NAMSIZ  = 20
 )
 
+const (
+	MIN_ZAPIVER uint8 = 2
+	MAX_ZAPIVER uint8 = 6
+)
```

Comment:
Can you use CamelCase instead SnakeCase?

Reply:
I'm sorry. I will change these variables.